src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public Rectangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg() {
+        return this.firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return this.secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return this.firstLeg * this.secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
```

Comment:
Avoid using `System.out.println` directly in a method of the class. It's better to return the string from `toString()` method and print it elsewhere, to follow the single responsibility principle.